src/handlers/scheduledEventHandler.ts

```diff
@@ -1,24 +1,26 @@
 import { env } from "../typeDefinitions/default.types";
 import { taskOverDueDiscordMembers } from "../utils/taskOverDueDiscordMembers";
-import * as error from "../constants/responses";
-import { getDiscordIds } from "../utils/getDiscordIds";
 import config from "../../config/config";
 import { SUPER_USER_ONE, SUPER_USER_TWO } from "../constants/variables";
 
 export async function send(env: env): Promise<void> {
   try {
-    const assigneeIds: string[] | string = await taskOverDueDiscordMembers();
+    let discordIds: string[] | string = await taskOverDueDiscordMembers();
 
-    //A user might have more than one task which are running red
-    //so to mention them just once, we are using Set to filter out
-    const discordIds: string[] | string = await getDiscordIds(assigneeIds);
-    const uniqueDiscordIds = [...new Set(discordIds)];
+    if (!Array.isArray(discordIds)) {
+      // If it's not an array, convert it to an array with a single element
+      discordIds = [discordIds];
+    }
+
+    if (discordIds.length === 0) {
+      return;
+    }
```

Comment:
Why will it not be an array?

Reply:
https://github.com/Real-Dev-Squad/discord-slash-commands/blob/develop/src/utils/taskOverDueDiscordMembers.ts#L9


Because of this utils function

Reply:
I have removed the extra type from this function.